pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -349,9 +353,15 @@ func evictPods(
 
 				keysAndValues := []interface{}{
 					"node", nodeInfo.node.Name,
-					"CPU", nodeInfo.usage[v1.ResourceCPU].MilliValue(),
-					"Mem", nodeInfo.usage[v1.ResourceMemory].Value(),
-					"Pods", nodeInfo.usage[v1.ResourcePods].Value(),
+				}
+				if quantity, exists := nodeInfo.usage[v1.ResourceCPU]; exists {
```

Comment:
Why do we use nodeInfo here? Can we us totalAvailableUsage variable instead? And extract this block for resolving keysAndValues for logging into a function?

Reply:
`totalAvailableUsage` includes available usage across all overutilized nodes. `nodeInfo.usage` is node specific.

Reply:
Ack, thanks for the explanation.